src/ui/mesh/qgsnewmeshlayerdialogbase.ui

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsNewMeshLayerDialogBase</class>
+ <widget class="QDialog" name="QgsNewMeshLayerDialogBase">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>477</width>
+    <height>479</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>New Mesh Layer</string>
+  </property>
+  <layout class="QFormLayout" name="formLayout">
+   <item row="0" column="0">
+    <widget class="QLabel" name="label_3">
+     <property name="text">
+      <string>Layer Name</string>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="1">
+    <widget class="QLineEdit" name="mLayerNameLineEdit">
+     <property name="text">
+      <string>New Mesh Layer</string>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="0">
+    <widget class="QLabel" name="label_2">
+     <property name="text">
+      <string>File Name</string>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="1">
+    <widget class="QgsFileWidget" name="mFileWidget" native="true"/>
+   </item>
+   <item row="2" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>File Format</string>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="1">
+    <widget class="QComboBox" name="mFormatComboBox"/>
+   </item>
+   <item row="3" column="1">
+    <widget class="QgsProjectionSelectionWidget" name="mProjectionSelectionWidget" native="true"/>
+   </item>
+   <item row="4" column="0" colspan="2">
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
+      <string>Frame of the New Mesh</string>
```

Comment:
I must admit I don't understand this string at all -- what does it mean?

Reply:
Frame is the faces/vertices of the mesh, without dataset. Here it is on which frame the new mesh will be based.

I confess I have some difficulties to find a good string ... Maybe "Base of the new mesh frame"

Reply:
What exactly does this setting do though? Does it copy the data from the selected layer into the new mesh?

Reply:
yes, but only the faces/vertices, not dataset. If user want to create and edit a new mesh, he can use an existing one.

Reply:
Hmm... Maybe "Initialize Mesh Using"?